package.json

```diff
@@ -75,9 +75,11 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
+    "clean": "lerna clean",
+    "lerna": "lerna",
```

Comment:
Added a generic `lerna` run-script so that any `lerna` commands can be easily run locally.